Lib/argparse.py

```diff
@@ -1268,8 +1268,14 @@ def __call__(self, string):
 
         # all other arguments are used as file names
         try:
-            return open(string, self._mode, self._bufsize, self._encoding,
+            file = open(string, self._mode, self._bufsize, self._encoding,
                         self._errors)
+
+            # Register cleanup function to close file
+            import atexit
```

Comment:
Please import modules at the top of the file.

Reply:
Oh sure! I had seen lazy imports throughout the codebase so thought this might have been a good candidate as well. Added `import atexit as _atexit` at the top 🙂